### ISSUE TITLE
ensime opt-out

### ISSUE THREAD

**Issue by fommil** [scalacenter/load-plugin#10]
Hi @Duhemm,

I'd like to opt-out of this approach being made available for people wanting to try out ensime. Please do not make it easy for people to install ensime this way.

The reason is that people who do not read the documentation at http://ensime.github.io/ (which is designed to explain the various moving parts) end up becoming a huge burden for us as volunteers. We have experienced this, it is not a theoretical problem. It frankly burnt me out: these people often think ENSIME is a commercial organisation and can be exceptionally demanding and rude.

Fundamentally, we do not want to appeal to the kind of user who "just wants it to work" we want to appeal to the kind of developer who relishes understanding how it works and how they can use ensime to customise and optimise their workflow.

In short, let people come to ensime in their own time. Please send them to some other editor as their first experience.

Best regards,
Sam

**Comment by Duhemm** [comment]
Hi @fommil, thanks for the heads up.

We don't recommend ensime to be used with this plugin, we only describe that dragos-vscode-scala uses it.
